pkg/subctl/cmd/join.go

```diff
@@ -74,7 +89,8 @@ func joinSubmarinerCluster(subctlData *datafile.SubctlData) {
 	panicOnError(err)
 
 	fmt.Printf("* Deploying Submariner\n")
-	err = deploy.Ensure(config, SubmarinerNamespace, repository, version, clusterID, serviceCIDR, clusterCIDR, subctlData)
+	err = deploy.Ensure(config, SubmarinerNamespace, repository, version,
```

Comment:
At some point I propose we turn this into a structure, it's growing out of control, lol :D

Reply:
I wonder what structure would be suitable ;-).